src/test/java/org/openmrs/contrib/qaframework/page/Attachmentspage.java

```diff
@@ -0,0 +1,52 @@
+package org.openmrs.contrib.qaframework.page;
+
+import java.util.List;
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class Attachmentspage extends Page {
+
+    public static final By ATTACHEMENT = By.id("attachments.attachments.visitActions.default");
+    public static final By CLICKORDROPAFILE = By.id("visit-documents-dropzone");
+    public static final By CAPTION = By.xpath("//*[@id=\"att-page-main\"]/div[1]/att-file-upload/div[2]/div/div[2]/textarea");
```

Comment:
the problem with xpath through divs is that when interface changes come in, these get broken, figure out how to use attributes if you can please